symfony/plugins/orangehrmTimePlugin/test/fixtures/testcases/MyTimesheetItemAPITestCases.yaml

```
@@ -284,7 +284,18 @@ GetAll:
         "middleName": "",
         "employeeId": "0024",
         "terminationId": null
-      }
+      },
+      "allowedActions": [
+        {
+          "action": "Submit",
+        },
```

```suggestion
        {
          "action": "SUBMIT",
          "name": "Submit"
        },
```
what do you think this object?

---

done